capstone/src/components/App/App.jsx

```diff
@@ -1,28 +1,56 @@
 import './App.css';
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import axios from 'axios';
 import Visitor from '../Visitor/Visitor';
-
+import User from '../User/User';
 // CONTEXTS
-// import LoginContext from '../Contexts/LoginContext';
+import UserContext from '../../Contexts/UserContext';
 // import SignUpContext from '../Contexts/SignUpContext';
 
+export async function getUserInfo(userId) {
+  const values = { objectId: userId };
+  const baseURL = 'http://localhost:3001';
+  return axios.post(`${baseURL}/visitor/user`, values);
+}
+
 export default function App() {
   // TODO: Handle state with useContext and useMemo
-  const [isLoggedIn, setIsLoggedIn] = useState(localStorage.getItem('current_user_id') !== null);
-
+  const [isLoggedIn, setIsLoggedIn] = useState(localStorage.getItem('current_user_id') != null);
```

Comment:
Can we clean up the render body of your App component a bit?

Right now, App has in its scope 1) where the user data info is stored (local storage) and 2) what key is used to get it (current_user_id). Does it really need this much info?

To make things easier in the future if we need to refactor stuff, we can loosely couple that data layer from the React layer by extracting it out of the component.

```
function getCurrentUserID() {
  return localStorage(...);
}

export default function App() {...}
```